salmon/commands.py

```diff
@@ -218,22 +218,22 @@ def checkconf(tracker, reset):
     Use the -r flag to reset/create the whole config file.
     """
     if reset:
-        click.secho("Resetting new config.py file", fg="cyan", bold=True)
-        
+        click.secho("Resetting new config.toml file", fg="cyan", bold=True)
+        # FIXME: need to keep track of path to config
```

Comment:
Multiple ways of tackling this, I just haven't written anything to do it yet.

Allowing both `XDG_CONFIG_HOME` *and* a local `config.toml` makes the logic slightly more complicated.